modules/kafka/src/main/java/org/apache/ignite/stream/kafka/connect/CacheEventFilter.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.stream.kafka.connect;
+
+import java.io.Serializable;
+import org.apache.ignite.events.CacheEvent;
+
+/**
+ * Filter for {@link IgniteSourceTask}.
+ * Enables filtering out cache events that do not match user-specified match conditions.
+ */
+public interface CacheEventFilter extends Serializable {
```

Comment:
Probably you can reuse org.apache.ignite.lang.IgnitePredicate there that was created for purposes like that one.